cli/config/configfile/file.go

```diff
@@ -48,6 +48,7 @@ type ConfigFile struct {
 	Experimental         string                      `json:"experimental,omitempty"`
 	StackOrchestrator    string                      `json:"stackOrchestrator,omitempty"`
 	Kubernetes           *KubernetesConfig           `json:"kubernetes,omitempty"`
+	CLIPluginExtraDirs   []string                    `json:"cli-plugin-extra-dirs,omitempty"`
```

Comment:
Should we keep camel case convention for the config file?

Reply:
CLI is an abbreviation so I think this is camel case. Spelling it out as `CommandLineInterfacePluginExtraDirs` would make my hands hurt.

That said, I don't much like the name, plus I wonder if this ought to be a top level cli plugins object with a sub object with the paths array (and potentially in the future other things).

Reply:
Sorry I was talking about the json interpretation which is used for the configuration file parsing.
```
`json:"cli-plugin-extra-dirs,omitempty"`
```

but yes I do agree camel case would make it look strange, but I feel the same way about having everything follow a convention and then one with its own.

Reply:
oh, I misunderstood, hadn't noticed that the json bits were camelly. I guess it should be `cliPluginsExtraDirs` then (`cLIPLuginsExtraDirs` doesn't seem right...)